fcos-graph-builder/src/scraper.rs

```diff
@@ -157,11 +168,11 @@ pub(crate) struct GetCachedGraph {
 }
 
 impl Message for GetCachedGraph {
-    type Result = Result<graph::Graph, Error>;
+    type Result = Result<Vec<u8>, Error>;
```

Comment:
This could be a [Bytes](https://docs.rs/bytes/1.1.0/bytes/) instance or so (i.e. refcounted to avoid copying) right?

Reply:
Yes, I think this would be better as some kind of RC (maybe not a full `Bytes` as we already get a `Vec` from serde and we can just directly refcount that). In fact, I started looking in that direction first and then realized the rework in this PR was needed before getting there.

Reply:
https://github.com/coreos/fedora-coreos-cincinnati/pull/62
`actix-web` has native support for the `bytes` crate, so I ended up using `Bytes` as you suggested (instead of plain `Vec` refcounting) in order to avoid some further copies down the road in the web framework/handler.